client/view/index/Index.html

```diff
@@ -133,13 +145,46 @@ <h2>Try an example deck</h2>
 
 export default {
 	data() {
+		var url = window.location.href.split("?");
+		if (url[1] !== undefined) window.location.replace("/#/home");
+		var searchParams = new URLSearchParams(url[1]);
+		var msgopt = "";
+
 		return {
 			sheetUrl: ``,
 			exampleWorkbooks,
+			message: searchParams.get("messagecontent"),
+			key: searchParams.get("key"),
 		}
 	},
 	components: {
 		SpreadsheetInput,
 	},
-}
+	methods: {
+		createNewSheet: function(key) {
+			const workbook = JSON.parse(sessionStorage.getItem(key.key))
+			this.mediatorCall(`rememberWorkbook`, workbook.key, workbook.name)
+			this.hideAlert();
+			const newWorkbook = `
+			<li>
+				<a href="/#/workbook/` + workbook.key + `">
+					` + workbook.name + `
+				</a>
+				(<a 
+					href="/#/settings/` + workbook.key + `"
+					style="font-style: italic;"
+				>more</a>)
+			</li>
+			`;
+			if (this.refs.review.innerHTML === "") {
+				this.refs.review.innerHTML = "<h2>Review one of your decks</h2><ul>" + newWorkbook + "</ul>"
```

Comment:
Our JS code should never, never manipulate the HTML in the DOM directly.  It is much simpler and efficient to let Svelte handle that for us.

The ideal is that the HTML-like syntax at the top of the component describes every possible way that the view could appear, and the JS code only changes the values in the state.  When the state changes, Svelte handles updating the live HTML for us, based on the template.